nuxeo-core/nuxeo-core-convert-api/src/main/java/org/nuxeo/ecm/core/convert/api/ConversionService.java

```diff
@@ -131,4 +147,11 @@ BlobHolder convertToMimeType(String destinationMimeType, BlobHolder blobHolder,
      * @since 7.4
      */
     BlobHolder getConversionResult(String id, boolean cleanTransientStoreEntry);
+
+    /**
+     * Adjusts the blob file name
+     *
+     * @since 9.2
+     */
+    void adjustBlobName(String filename, Blob blob, String extension, String mimeType);
```

Comment:
You shouldn't need this method in this interface.
Usage are:
- in a test method, instantiate the variable as `ConversionServiceImpl` and call directly the protected method 
- in an operation, but not sure if it's needed